dynamodb-stream-eventbridge-fanout/src/main/java/no/unit/nva/doi/lambda/FanoutHandler.java

```diff
@@ -0,0 +1,82 @@
+package no.unit.nva.doi.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
+import java.time.Duration;
+import no.unit.nva.doi.publisher.EventBridgePublisher;
+import no.unit.nva.doi.publisher.EventBridgeRetryClient;
+import no.unit.nva.doi.publisher.EventPublisher;
+import no.unit.nva.doi.publisher.SqsEventPublisher;
+import no.unit.nva.doi.utils.JacocoGenerated;
+import software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider;
+import software.amazon.awssdk.core.client.config.ClientOverrideConfiguration;
+import software.amazon.awssdk.core.retry.RetryPolicy;
+import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
+import software.amazon.awssdk.services.sqs.SqsClient;
+
+public class FanoutHandler implements RequestHandler<DynamodbEvent, Void> {
+
+    public static final String AWS_REGION = "AWS_REGION";
+    private final EventPublisher eventPublisher;
+
+    @JacocoGenerated
+    public FanoutHandler() {
+        this(defaultEventBridgePublisher());
+    }
+
+    @JacocoGenerated
+    private static EventPublisher defaultEventBridgePublisher() {
+        return new EventBridgePublisher(
+            defaultEventBridgeRetryClient(),
+            defaultFailedEventPublisher(),
+            Env.getEventBusName()
+        );
+    }
+
+    @JacocoGenerated
+    private static EventPublisher defaultFailedEventPublisher() {
+        return new SqsEventPublisher(defaultSqsClient(), Env.getDlqUrl());
+    }
+
+    @JacocoGenerated
+    private static SqsClient defaultSqsClient() {
+        return SqsClient.builder()
```

Comment:
to my understanding if we set up DLQ or multiple destinations we shouldn't post things directly. Lambda should post things In DLQ automatically. Otherwise, what is the point of declaring in the template?